pages/common/subl.md

```diff
@@ -0,0 +1,19 @@
+# subl
+
+> Sublime Text, a sophisticated text editor for code, markup and prose.
+
+- Open a file or folder:
+
+`subl {{path/to/file_or_folder}}`
+
+- Open a file or folder in a new window:
+
+`subl -n {{path/to/file_or_folder}}`
+
+- Add a folder to existing window:
+
+`subl -a {{path/to/folder}}`
+
+- Open a Project:
+
+`subl --project {{path/to/project/file}}`
```

Comment:
Do project files have a specific extension? Perhaps we could make it `{{path/to/project/file.ext}}`, where `ext` is the preferred file extension for sublime text project files.

Reply:
According to the docs, it looks like there are 2 files extensions.
`.sublime-project` - the project definition
`.sublime-workspace` - user specific data

I'm assuming that this command would use the first one.

Reply:
Yeah!